app/src/test/java/com/yalta/repositories/FakeLocationRepo.kt

```diff
@@ -0,0 +1,21 @@
+package com.yalta.repositories
+
+import android.location.Location
+import java.sql.Timestamp
+
+class FakeLocationRepo : LocationRepo {
+    private var locations: MutableList<Pair<Double, Double>> = mutableListOf()
+
+    override suspend fun sendCurrentLocation(location: Location): LocationResponse {
```

Comment:
maybe `update...`?

Reply:
But
It's not update, it's literally sending current location...